catroid/src/main/java/org/catrobat/catroid/ui/fragment/FormulaEditorFragment.java

```diff
@@ -630,6 +630,7 @@ public boolean saveFormulaIfPossible() {
 				if (Brick.BrickField.isExpectingStringValue(currentBrickField)) {
 					return saveValidFormula(new FormulaElement(FormulaElement.ElementType.STRING, "", null));
 				}
+				// fallthrough
```

Comment:
@rschluesselbauer, @DoenerMaschine, @robotic-heidenreich
Please review if this fallthrough is on purpose (https://github.com/Catrobat/Catroid/pull/2269)